src/main/java/bq_standard/tasks/TaskCrafting.java

```diff
@@ -80,10 +81,10 @@ public void detect(ParticipantInfo pInfo, DBEntry<IQuest> quest)
 	    pInfo.markDirtyParty(Collections.singletonList(quest.getID()));
 	}
 	
-	public void onItemCraft(ParticipantInfo pInfo, DBEntry<IQuest> quest, ItemStack stack)
+	public void onItemCraft(ParticipantInfo pInfo, DBEntry<IQuest> quest, ItemStack stack, IntSupplier realStackSizeSupplier)
```

Comment:
I assume we're purposefully not preserving the old public signature here? We don't expect anyone else to be calling this I suppose?